src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -142,55 +141,54 @@ func (s *Service) cmdGithubCreate(roomID, userID string, args []string) (interfa
 	return gomatrix.TextMessage{"m.notice", fmt.Sprintf("Created issue: %s", *issue.HTMLURL)}, nil
 }
 
-func (s *Service) cmdGithubComment(roomID, userID string, args []string) (interface{}, error) {
+const cmdGithubReactUsage = `!github react [owner/repo]#issue (+1|-1|laugh|confused|heart|hooray)`
```

Comment:
Added their formal names, their formal names wrapped in `:`'s and their emoji variants as aliases

PTAL @ara4n